src/test/java/com/datadog/api/TestClient.java

```diff
@@ -285,7 +298,15 @@ public Response method(String s, Entity<?> entity) {
       if (requestBody == null) {
         assertEquals("", entity.getEntity());
       } else {
-        String json = (String) requestBody.get("json");
+        String json;
+        try {
+          json = (String) requestBody.get("json");
+        } catch (ClassCastException e) {
+          json =
+              new Gson()
+                  .toJson((LinkedHashMap) requestBody.get("json"), LinkedHashMap.class)
+                  .toString();
+        }
```

Comment:
```suggestion
        String json = getBody(requestBody);
```
Can we do the same here?